public/static/docs/command-reference/get.md

```diff
@@ -12,17 +12,18 @@ Download a file or directory from any <abbr>DVC project</abbr> or Git repository
 usage: dvc get [-h] [-q | -v] [-o [OUT]] [--rev [REV]] url path
 
 positional arguments:
-  url         URL of Git repository with DVC project to download from.
-  path        Path to a file or directory within a DVC repository.
+  url                   Location of DVC project or Git repository to download
```

Comment:
minor: please keep a single line here and below

Reply:
Yes @fabiosantoscode, just remove space tabs between `url` and the description as the previous version to keep these to a single line when possible. It should be possible in this case.

Reply:
Got it

Reply:
I see this is fixed. Thanks!